notifications/NotificationPushsafer.cpp

```diff
@@ -4,10 +4,22 @@
 #include "../main/Logger.h"
 #include "../httpclient/UrlEncode.h"
 
+#include <stdio.h>
+#include "../webserver/Base64.h"
+#include <curl/curl.h>

```

Comment:
Please use our HTTPClient class for downloading files instead of creating a new curl method